compensation-benefits.md

```diff
@@ -0,0 +1,35 @@
+# Compensation and Benefits
+
+In December 2021 we revised our total compensation and switched to a salaried model. At that time we polled member-workers on what types of [benefits they want](https://docs.google.com/document/d/1_jQ8b15P5KrqxPI9KEVWIgLOlG5HDE5xBazd5Ncfd8U/edit) 🔒 and a decision was made to implement health benefits as many were identified as a priority, detailed below.
+
+## Compensation
+
+Employees at Hypha, full-time or part-time, permanent or otherwise, are evaluated based on the following criteria:
+
+* skills and experiences
+* responsibilities
+* seniority in organization
+
+Full details and salary tiers are detailed in our [Employee Salary Guide](salary.md).
+
+## Health Benefits
+
+Our provider is the Co-operators and we are able to provide the following to members based on hours worked:
+
+| Employee Status           | Benefits                             |
+|---------------------------|--------------------------------------|
+| PT (0.25 or 1 day/week )  | No benefits                          | 
+| PT (0.50 or 2 days/week)  | Life, AD&D, LTD + HSA                |
+| FT (above 0.50)           | Life, AD&D, LTD, EHC, Dental + HSA   |
```

Comment:
@dcwalk the wording in the contract I think is:
- 20h/week = FT
- 15h/week = PT

Since we only deal in days/week, we should:

```suggestion
| FT (0.75 or 3 days/week, and above) | Life, AD&D, LTD, EHC, Dental + HSA   |
```